.gitignore

```diff
@@ -9,4 +9,6 @@
  
 # Maven
 log/
-target/
\ No newline at end of file
+target/
+
+.jar
```

Comment:
IDE specific files could be included into commit. I do not see in the `.gitignore` lines which block commits project related files